chia/_tests/pools/test_pool_cmdline.py

```diff
@@ -1,20 +1,1050 @@
+from chia.wallet.wallet_state_manager import WalletStateManager
+
+
+@dataclass
+class StateUrlCase:
+    id: str
+    state: str
+    pool_url: Optional[str]
+    expected_error: Optional[str] = None
+    marks: Marks = ()
+
+
+async def verify_pool_state(wallet_rpc: WalletRpcClient, w_id: int, expected_state: PoolSingletonState) -> bool:
+    pw_status: PoolWalletInfo = (await wallet_rpc.pw_status(w_id))[0]
+    return pw_status.current.state == expected_state.value
+
+
+async def process_first_plotnft_create(
+    wallet_test_framework: WalletTestFramework, expected_state: PoolSingletonState
+) -> int:
+    wallet_rpc: WalletRpcClient = wallet_test_framework.environments[0].rpc_client
+
+    await wallet_test_framework.process_pending_states(
+        [
+            WalletStateTransition(
+                pre_block_balance_updates={
+                    1: {
+                        "confirmed_wallet_balance": 0,
+                        "unconfirmed_wallet_balance": -1,
+                        "<=#spendable_balance": 1,
+                        "<=#max_send_amount": 1,
+                        ">=#pending_change": 1,  # any amount increase
+                        "pending_coin_removal_count": 1,
+                    },
+                },
+                post_block_balance_updates={
+                    1: {
+                        "confirmed_wallet_balance": -1,
+                        "unconfirmed_wallet_balance": 0,
+                        ">=#spendable_balance": 1,
+                        ">=#max_send_amount": 1,
+                        "<=#pending_change": 1,  # any amount decrease
+                        "<=#pending_coin_removal_count": 1,
+                    },
+                    2: {"init": True, "unspent_coin_count": 1},
+                },
+            )
+        ]
+    )
+
+    summaries_response = await wallet_rpc.get_wallets(WalletType.POOLING_WALLET)
+    assert len(summaries_response) == 1
+    wallet_id: int = summaries_response[-1]["id"]
+
+    await verify_pool_state(wallet_rpc, wallet_id, expected_state=expected_state)
+    return wallet_id
+
+
+async def process_second_plotnft_create(
+    wallet_test_framework: WalletTestFramework,
+    expected_state: PoolSingletonState,
+) -> int:
+    wallet_rpc: WalletRpcClient = wallet_test_framework.environments[0].rpc_client
+
+    await wallet_test_framework.process_pending_states(
+        [
+            WalletStateTransition(
+                pre_block_balance_updates={
+                    1: {
+                        "confirmed_wallet_balance": 0,
+                        "unconfirmed_wallet_balance": -1,
+                        "<=#spendable_balance": 1,
+                        "<=#max_send_amount": 1,
+                        ">=#pending_change": 1,  # any amount increase
+                        "pending_coin_removal_count": 1,
+                    },
+                    2: {
+                        "set_remainder": True,
```

Comment:
Is this (and the one below) really a good idea?  I would imagine this wallet is just not doing anything since the new one is being created so I wouldn't expect there to be any balance changes to have to yada-yada away.

Reply:
I think this is a good use. When creating a pool wallet, nothing of interest should be happening in other pool wallets and so I like just wholesale ignore. I haven't fully internalized this balance update checker and I'm pretty sure this was throwing errors that are unrelated to created the next plotnft

Reply:
Nothing of interest happening is something that should be asserted is my point.  If something _is_ happening that shouldn't be, this will ignore that buggy behavior.